### PR TITLE
Bug 1549637 - Homepage link on the global header is wrong

### PR DESCRIPTION
**Pull request by dylanwh**

### CODE REVIEW

File: template/en/default/search/form.html.tmpl [hunk 1]
```diff
@@ -43,10 +43,10 @@ var tms = new Array();
 [%# Create an array of products, indexed by the classification #%]
 
 [% nclass = 0 %]
-[% FOREACH c = classification %]
+[% FOREACH classification = classification %]
```
Comment (kyoshino) [inline template/en/default/search/form.html.tmpl]: Hooray TT. Looks like this change introduces another issue 😅 The Classification list will be empty on the Advanced Search page. `cls` or whatever should work.

Comment (dylanwh) [reply template/en/default/search/form.html.tmpl]: damnit TT

Comment (kyoshino) [reply template/en/default/search/form.html.tmpl]: _/me awaiting an update..._

**Comment by dklawren** [comment]
Seems like it would have been easier just to change the mojo variable to something else?

 $params{self} = $params{controller} = $c;

**Comment by dylanwh** [comment]
> Seems like it would have been easier just to change the mojo variable to something else?
> 
> $params{self} = $params{controller} = $c;

I wanted to be consistent with existing mojo codebases.

**Pull request merged**